sdk/cosmosdb/cosmos/samples/v3/javascript/README.md

```diff
@@ -0,0 +1,86 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure-cosmos-db
+urlFragment: cosmos-javascript
+---
+
+# Azure Cosmos DB client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure Cosmos DB in some common scenarios.
+
+| **File Name**                                                                 | **Description**                                                                                     |
+| ----------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| [typescript/Shared/handleError.js][typescript_shared_handleerror]             | Demonstrates using SasTokens for granting scoped access to Cosmos resources. _Private feature_      |
```

Comment:
I think what we want at the end of the day is for this Shared/handleError.js file to end up hidden (`@azsdk-util true` mentioned above should take care of this) and for the `typescript/src` paths to be omitted from the following files, which should be taken care of if we can flatten the sample directory hierarchy.